### DISCUSSION TITLE
Blocking page generation is triggered by link prefetching

### DISCUSSION THREAD

**Discussion by Maniload** [vercel/next.js#41740] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Home
    Binaries:
      Node: 16.15.1
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 12.3.2-canary.33
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0

### How are you deploying your application? (if relevant)

next start

### Describe the Bug

When I use `fallback: "blocking"` in combination with `<Link>` pretching, I get huge amounts of page builds (and therefore API requests on the server side) after deploying because I have index pages with lots of links.

### Expected Behavior

Pages should only get built after a user clicks on a link.

### Link to reproduction

https://github.com/ManuelDevelopment/prefetch-blocking-page-build

### To Reproduce

1. Run `npm run build` and `npm run start`
2. Visit the index page
3. There should be five console log messages on the server, one for every page build

**Comment by balazsorban44** [comment]
The linked reproduction seemed to be empty, but this should be expected recently. The default behavior is to prefetch pages that have a `Link` in the viewport. To only prefetch when a link is hovered, you can set `prefetch={false}` on the links.

See the docs: https://nextjs.org/docs/api-reference/next/link

**Reply by Maniload** [reply]
Oh sorry, I somehow created the repository but didn't push my changes. It should be updated now.

I'm aware of the prefetch functionality, because I already had a similar problem that was fixed here: https://github.com/vercel/next.js/issues/17758. This issue is basically the same, but for the initial (blocking) page generation. The workaround with `prefetch={false}` was also suggested in that issue, but later it got fixed by deactivating revalidation when it's triggered by prefetching.

So my question is, if we can get the same behavior for pages that don't exist yet?
